upgrade/README.md

```diff
@@ -9,11 +9,10 @@ product streams for the HPE Cray EX system can be installed or upgraded.
 - [Upgrade CSM](#upgrade-csm)
   - [Known Issues](#known-issues)
   - [1. Prepare for upgrade](#1-prepare-for-upgrade)
-  - [2. (Optional) Enable Customer High Speed Network Routing](#2-optional-enable-customer-high-speed-network-routing)
-  - [3. Upgrade management nodes and CSM services](#3-upgrade-management-nodes-and-csm-services)
-  - [4. Validate CSM health](#4-validate-csm-health)
-  - [5. Apply security hardening](#5-apply-security-hardening)
-  - [6. Next topic](#6-next-topic)
+  - [2. Upgrade management nodes and CSM services](#2-upgrade-management-nodes-and-csm-services)
```

Comment:
The broken links into this page also need to be addressed. This PR only fixes the changes within this page itself, but does not cover the links into this page from other pages. This is described as part of [CASMINST-5479](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5479). In that ticket I listed the two cases I found of that, but there may be others.

Reply:
markdown link test is passing, assume this is resolved.

Reply:
It is not resolved -- the markdown link checker does not check links to specific sections, unfortunately. It only makes sure that the target document exists.